flutter/lib/screens/offline_screen.dart

```diff
@@ -0,0 +1,77 @@
+//Flutter packages
+import 'dart:async';
+import 'package:flutter/material.dart';
+//Third party packages
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:lottie/lottie.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:simple_connection_checker/simple_connection_checker.dart';
+//My Packages
+import '../screens/home_screen.dart';
+import '../values.dart' as values;
+
+class OfflineScreen extends StatefulWidget {
+
+  static const routeName = "/offline_screen";
+
+  const OfflineScreen({Key? key}) : super(key: key);
+
+  @override
+  _OfflineScreenState createState() => _OfflineScreenState();
+}
+
+class _OfflineScreenState extends State<OfflineScreen> {
+
+  //Object declarations
+
+  String _seed = "";
+
+  //Offline handler
+  StreamSubscription? _offlineStreamSubscription;
```

Comment:
**suggestion:** instead of then defining another listener, this would already be handled in the home screen and render the appropriate content.